### PR TITLE
B 18638 create ww order type

### PR DESCRIPTION
**Pull request by loganwc**
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A904632&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

[INT PR](https://github.com/transcom/mymove/pull/12225)
[INT PR 2](https://github.com/transcom/mymove/pull/12277)

### CODE REVIEW

File: pkg/handlers/ghcapi/internal/payloads/model_to_payload.go [hunk 1]
```diff
@@ -1903,7 +1903,7 @@ func SearchMoves(appCtx appcontext.AppContext, moves models.Moves) *ghcmessages.
 			ShipmentsCount:                    int64(numShipments),
 			OriginDutyLocationPostalCode:      move.Orders.OriginDutyLocation.Address.PostalCode,
 			DestinationDutyLocationPostalCode: move.Orders.NewDutyLocation.Address.PostalCode,
-			OrderType:                         string(move.Orders.OrdersType),
+			OrderType:                         (*string)(move.Orders.OrdersType.Pointer()),
```
Comment (paulstonebraker) [inline pkg/handlers/ghcapi/internal/payloads/model_to_payload.go:1906]: I don't see this change on INT

File: src/containers/Headers/CustomerLoggedInHeader.jsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const CustomerLoggedInHeader = ({ state, isProfileComplete, logOut }) => {
   };
 
   return (
-    <MilMoveHeader isSpecialMove={isSpecialMove}>
+    <MilMoveHeader specialOrderType={specialOrderType}>
       <CustomerUserInfo showProfileLink={isProfileComplete} handleLogout={handleLogout} />
```
Comment (paulstonebraker) [inline src/containers/Headers/CustomerLoggedInHeader.jsx:39]: INT PR has moveId={moveId}

File: src/pages/MyMove/Home/MoveHome.jsx [hunk 3]
```diff
@@ -409,7 +409,6 @@ const MoveHome = ({ serviceMemberMoves, isProfileComplete, serviceMember, signed
   // eslint-disable-next-line camelcase
   const currentLocation = current_location;
   const shipmentNumbersByType = {};
-
   return (
```
Comment (paulstonebraker) [inline src/pages/MyMove/Home/MoveHome.jsx:412]: INT has const isSpecialMove = CHECK_SPECIAL_ORDERS_TYPES(orders?.orders_type); and <p>{SPECIAL_ORDERS_TYPES[`${orders.orders_type}`]}</p> under line 426

File: swagger-def/ghc.yaml [hunk 4]
```diff
@@ -5161,6 +5162,7 @@ definitions:
         $ref: '#/definitions/GBLOC'
       orderType:
         type: string
+        x-nullable: true
```
Comment (paulstonebraker) [inline swagger-def/ghc.yaml:5165]: don't see this on INT

File: src/containers/Headers/CustomerLoggedInHeader.jsx (hunk 2 above)
Comment (loganwc) [reply src/containers/Headers/CustomerLoggedInHeader.jsx:39]: moveid is part of another PR

File: src/pages/MyMove/Home/MoveHome.jsx (hunk 3 above)
Comment (loganwc) [reply src/pages/MyMove/Home/MoveHome.jsx:412]: Those changes were removed in https://github.com/transcom/mymove/pull/12254/ not sure why that's not reflecting in the PR though

**Review by pambecker — APPROVED** [review]

**Review by paulstonebraker — APPROVED** [review]

**Pull request merged**